deps/wazuh_testing/wazuh_testing/end_to_end/monitoring.py

```diff
@@ -95,10 +97,15 @@ def filter_events_by_timestamp(match_events: List) -> List:
                 timestamp_format = "%Y/%m/%d %H:%M:%S"
                 timestamp_format_parameter = "%Y-%m-%dT%H:%M:%S.%f"
 
-                timestamp_datetime = datetime.strptime(timestamp_str, timestamp_format)
-                greater_than_timestamp_formatted = datetime.strptime(greater_than_timestamp, timestamp_format_parameter)
+                try:
+                    timestamp_datetime = datetime.strptime(timestamp_str, timestamp_format)
+                    greater_than_timestamp_formatted = datetime.strptime(greater_than_timestamp,
+                                                                         timestamp_format_parameter)
+                except ValueError:
+                    raise ValueError(f"Timestamp format not supported: {timestamp_str}."
+                                     'Do the regex includes the timestamp?')
```

Comment:
If you already have the `greater_than_timestamp` field, is it necessary to add the timestamp regex to the `regex` field? Isn't it a bit repetitive?

Reply:
To effectively compare with the time specified in the `greater_than_timestamp` parameters, we need to gather the timestamp from the event. Therefore, the specified regex should be designed to match at least one group, specifically tailored for capturing the timestamp.

Reply:
We will revisit this in the future